.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       matrix:
         architecture: ["arm64", "amd64"]
-        image: ["wolfi-base", "rocky9.2-10-gpu", "rocky9.2-10-slim", "rocky9.2-10-cpu"]
+        image: ["wolfi-base", "rocky9.2-gpu", "rocky9.2-slim", "rocky9.2-cpu"]
```

Comment:
Think the rockylinux directories under `dockerfiles` will need to change to drop the `-10` as well.